src/http2_connection.erl

```diff
@@ -3,7 +3,9 @@
 -behaviour(gen_fsm).
 
 -include("http2.hrl").
+-include("h2_streams.hrl").
```

Comment:
I need the records in two places for pattern matching. Really, I think of h2_streams as something that could be in http2_connection, but that I could put in another file for easier reading.